example/android/app/src/main/AndroidManifest.xml

```diff
@@ -7,6 +7,7 @@
     <uses-permission android:name="android.permission.READ_MEDIA_IMAGES" />
     <uses-permission android:name="android.permission.READ_MEDIA_VIDEO" />
     <uses-permission android:name="android.permission.READ_MEDIA_AUDIO" />
+    <uses-permission android:name="android.permission.READ_MEDIA_VISUAL_USER_SELECTED" />
```

Comment:
This should be addressed in the READMEs too.

Reply:
From my testing it's optional, but I think it does state it in the README

Reply:
I don't see anything in the READMEs. 🤔

Reply:
I mean, it really should be stated in the README that it's optional.

In addition, when I say optional, I do not mean that the permission itself is optional, but "even if it is not added to the manifest, it can be used in the code."